Error: An accessible description was not requested by the prompt
diff --git a/output/Text Field/5/c321fa6b-93ec-43da-bdcb-cf066b3da02f.html b/output/Text Field/5/c321fa6b-93ec-43da-bdcb-cf066b3da02f.html
index ac9682a..994e005 100644
--- a/output/Text Field/5/c321fa6b-93ec-43da-bdcb-cf066b3da02f.html	
+++ b/output/Text Field/5/c321fa6b-93ec-43da-bdcb-cf066b3da02f.html	
@@ -1,3 +1,2 @@
 <label for="orange-color">What color is an orange?</label>
-<input type="text" id="orange-color" name="orange-color" aria-describedby="orange-hint">
-<span id="orange-hint">Hint: The color is in the name of the fruit.</span>
\ No newline at end of file
+<input type="text" id="orange-color" name="orange-color">
\ No newline at end of file
